.golangci.yaml

```diff
@@ -41,7 +40,15 @@ linters-settings:
       - Client
       - record.EventRecorder
       - client.Object
+  wrapcheck:
+    ignorePackageGlobs:
+      - github.com/kyma-project/lifecycle-manager/pkg/*
```

Comment:
If you ignore `pkg`, basically the meaning of this refactoring task is lost, right? most of the unwrapped errors come from pkg.

Reply:
@lindnerby is working in parallel to tackle the `pkg/` package. He will commit the new changes soon!

Reply:
this will ignore not wrapped errors that come from github.com/kyma-project/lifecycle-manager/pkg/*
since these are our pkgs, the consumer of this package (also us, controller most likely) doesn't need to always wrap them. but it is still analysing every error within pkg.